src/groupeddataframe/groupeddataframe.jl

```diff
@@ -316,6 +316,12 @@ end
 
 Base.to_index(gd::GroupedDataFrame, key::Tuple) = gd.keymap[key]
 
+Base.to_index(gd::GroupedDataFrame, key::Pair) = 
```

Comment:
We do not allow this form for other styles, but probably it is OK to allow this, as grouping by a single column is pretty common.

Reply:
Ah - now I remember why we disallow it - it will be ambiguous if `AbstractVector{<:Pair}` should select one cell based on many pairs or it should select multiple cells based on a single pair each.

Reply:
I am fine if we only allow vectors of `Pairs`.